src/index-debug.html

```diff
@@ -26,6 +26,9 @@
     <meta charset="utf-8">
     <title>Photoshop Design Space (development build)</title>
     <link rel="stylesheet" type="text/css" href="style.css" />
+    <script type="text/javascript">
+        var loadTime = new Date();
```

Comment:
oh, the top-level declarations. They will become global variables, unless in `strict mode`.